setup.py

```diff
@@ -52,7 +52,7 @@ def read(*names, **kwargs):
 
 setup(
     name='fast-numpy-loops',
-    version='0.0.0',
+    version='v0.0.2',
```

Comment:
This is inconsistent. It would be best to template this so there is one source of truth.

Reply:
Thomas had cut a real release with the name v0.0.1 so I thought that's what direction we should go. While testing, we published test releases without the v as a reminder to me that those needed to be removed. Also, I think the cookiecutter template may have had this without the v originally.